README.md

```diff
@@ -226,6 +226,10 @@ TypeScript may add an `export {};` if all `import`s and `export`s were removed (
 
 Because `ts-blank-space` only removes code, this is not performed. To force the output to always have an ESM syntactic marker, you can manually append `"export {};"`;
 
+## Looking into a bundler integration?
+
+There is currently a third-party module providing integration with Webpack & Rspack - Find more about [ts-blank-loader](https://github.com/leimonio/ts-blank-loader)
```

Comment:
Might be good to make this section more generic, so it's easier to add any other packages in the future.

```suggestion
## 3rd party ecosystem plugins

- Webpack/Rspack: [ts-blank-loader](https://github.com/leimonio/ts-blank-loader)
```

Could we also link to the section from the [contents](https://github.com/bloomberg/ts-blank-space?tab=readme-ov-file#contents). Thanks!